src/components/Field/Field.scss

```diff
@@ -42,10 +42,14 @@
   min-height: 80px;
   text-align: right;
   padding: 0;
+  background: #f6fafb;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
 .input {
-  margin: 25px 20px 0 0;
+  // margin: 25px 20px 0 0;
```

Comment:
This can be removed I guess

Reply:
remove the line or un-comment it ?

Reply:
Do you think we should keep it as a comment?

Reply:
I think removing them would be good.

Reply:
Done! https://github.com/parse-community/parse-dashboard/pull/1708/commits/16010c7db70c04f249edafd95bf313fe8a195461